src/scripts/ping-okta-server.ts

```diff
@@ -0,0 +1,117 @@
+import fetch from 'node-fetch';
+import * as TE from 'fp-ts/lib/TaskEither';
+import * as E from 'fp-ts/lib/Either';
+import { RootCommand } from '..';
+import * as Console from 'fp-ts/lib/Console';
+import { pipe } from 'fp-ts/lib/function';
+import { Argv } from 'yargs';
+
+export const pingOktaServer = async (
```

Comment:
**nit:** document this. What is the `string` that is returned here?

Reply:
modified in commit: https://github.com/agiledigital-labs/oktagon/commit/9f846a4fc822510fa69e963a39aa76a76ddec247